File: atinject-integration/src/main/resources/webapp/scripts/atinject/registration/RegisterAsGuestResponse.js
define(["atinject/WebSocketResponse", "atinject/user/User"],
function(WebSocketResponse, User){
    function RegisterAsGuestResponse() {
        this._class = "org.atinject.api.registration.dto.RegisterAsGuestResponse"
    };

    RegisterAsGuestResponse.prototype = new WebSocketResponse();

        RegisterAsGuestResponse.prototype.getUser = function() {
        return this.user;
    };

    RegisterAsGuestResponse.prototype.setUser = function(user) {
        this.user = user;
        return this;
    };


    
    return RegisterAsGuestResponse;
});

File: atinject-api/atinject-api-user-registration/src/main/java/org/atinject/api/registration/dto/RegisterAsGuestResponse.java
package org.atinject.api.registration.dto;

import org.atinject.api.user.dto.User;
import org.atinject.core.websocket.dto.WebSocketResponse;

public class RegisterAsGuestResponse extends WebSocketResponse {

    private static final long serialVersionUID = 1L;
    
    private User user;

    public RegisterAsGuestResponse(){
        super();
    }

    public User getUser() {
        return user;
    }

    public RegisterAsGuestResponse setUser(User user) {
        this.user = user;
        return this;
    }
    
}
